server/src/routers/converterRouter.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+const router = new express.Router();
+
+const { mapping } = require('../constants/config');
+const { parseInputFile, sendResponse } = require('../util');
+
+router.use(express.json());
+
+var Converter = require('api-spec-converter');
+
+//Endpoint for conversion
+router.post('/api/specification/:format/:convertTo', upload.single('file'), (req, res) => {
+    const {format, convertTo} = req.params,
+        {targetSyntax = 'json', toFile = false} = req.query;
+
+    // Check if the conversion is possible
+    if(mapping[format] && mapping[format].includes(convertTo)) {
+      const file = req.file;
+
+      if(!file) {
+        res.status(400).send({'message': 'File not available.'});
+      }
+
+      //Parse JSON / YAML Input String
+      const parsedSpec = parseInputFile(file);
+      if(!parsedSpec) {
+        return res.status(400).send({'message': 'File extension not available.'});
+      }
+      else if(parsedSpec === 'invalid') {
+        return res.status(400).send({'message': 'File extension not supported.'});
```

Comment:
Can you move the exit conditions/guard clauses to the top instead of bottom? It makes the code far easier to read. 
Read more here - https://qvault.io/clean-code/guard-clauses/
#CleanCode
@pcgamer1

Reply:
I think you've done it everywhere except the last one - else {
        return res.status(404).send({'message': 'Conversion mapping not found.'});
    }

so you can check that first and return this, then you don't need to have an else